modules/apigee/main.tf

```diff
@@ -137,31 +137,31 @@ resource "google_apigee_addons_config" "addons_config" {
   org      = local.org_name
   addons_config {
     dynamic "advanced_api_ops_config" {
-      for_each = var.addons_config.advanced_api_ops ? [] : [""]
+      for_each = var.addons_config.advanced_api_ops ? [""] : []
       content {
         enabled = true
```

Comment:
can we us the actual value here? 

The following should be enough:
```hcl
      advanced_api_ops_config {
        enabled = var.addons_config.advanced_api_ops
      }
```

And same for all the other options.

Reply:
Sure thing. 

Much cleaner like you suggested 👍